templates/base.html

```diff
@@ -65,11 +65,25 @@
 
 		<footer class="footer">
 			<div class="container">
-				<a href="https://github.com/CodeCatz">
-					<img alt="CodeCatz" src="{% static 'img/CodeCatz.jpg' %}">
-					<p>GITHUB CodeCatz: Kittens learning to code like Catz</p>
-				</a>
-			</div>
+				<div id="footrow">
+					<div class="col-md-4">
+						<ul class="ulfooter">
+							<li class="lifooter"><a href="{% url 'pages-about' %}">Sponsors</a></li>
+							<li class="lifooter"><a href="{% url 'pages-about' %}">Privacy</a></li>
+							<li class="lifooter"><a href="{% url 'pages-about' %}">Support</a></li>
+						</ul>
+					</div> <!-- end col-md-5 -->
```

Comment:
These comments don't seem to match the actual classes that are in use (for instance, in line 75, the div closes the div with the class col-md-4, not cold-md-5 - and these classes don't provide a unique identifier for the file). At this point, the code isn't too complicated to require a comment for each div, so you might actually leave them out.